src/app/config/config.ts

```diff
@@ -1,3 +1,13 @@
+// On bootstrapping the application, deliberately
+// import dotenv first and invoke it, so that we
+// pick up all env vars from .env before configuration
+
+/* eslint-disable import/first */
+// eslint-disable-next-line simple-import-sort/imports
+import dotenv from 'dotenv'
+
+dotenv.config()
+
```

Comment:
how do you feel about using the [preload](https://github.com/motdotla/dotenv#preload) feature instead of polluting the application code?